src/RBF/rbf.cpp

```diff
@@ -644,7 +644,7 @@ int RBFKernel::solve()
     int lda     = nS;
     int ldb     = nS;
     int info;
-    int ipiv[nS];
+    int *ipiv = (int*) malloc( nS * sizeof(int));
```

Comment:
If possible, I would use an std::vector (also in the POD module). Using malloc may lead to leaks. For example, after this change, ipiv is leaked if LAPACKE_dgesv fails (at line 678 there is a return before freeing ipiv).

Reply:
Sure, on it

Reply:
Handled